rmgpy/molecule/atomtype.py

```diff
@@ -240,20 +240,56 @@ def get_features(self):
 
 ATOMTYPES = {}
 
+ATOMTYPES['Rx']    = AtomType(label='Rx', generic=[], specific=[
```

Comment:
Can you run the changes through a formatter

Reply:
I've made the changes I think you're looking at, at least.